doc/man1/flux-resource.rst

```diff
@@ -396,6 +396,12 @@ Watch the resource journal, which is described in RFC 44.
   The current set of valid events events is *restart*, *resource-define*,
   *online*, *offline*, *drain*, *undrain*, *torpid*, and *lively*.
 
+.. option:: -i, --include=TARGETS
+
+  Filter events to only include those that apply to  *TARGETS*, which may be
+  specified either as an idset of broker ranks or list of hosts or nodes in
+  hostlist form.
+
```

Comment:
Might want to indicate a preference for hosts, given the caveat about historical reranking.  Maybe
> , which is normally one or more hostnames in RFC 29 hostlist form.  *TARGETS* can also be broker ranks in RFC 22 idset form, useful mainly in test.

Reply:
Fixed that and will set MWP.